### ISSUE TITLE
Mac OS X icon

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#2254]
**[Submitted to the original trac issue database at 6.11pm, Thursday, 3rd September 2009]**

I have created an icon for Mac OS X from the Icons/Mercator_splash.xcf file. To use it this should be added:

macx {
	RC_FILE = ../Icons/merkaartor.icns
}

to one of the project .pro or .pri files (src.pro perhaps).

**Comment by tomhughes** [comment]
**Author: aleksandar.topuzovic@gmail.com**
**[Added to the original trac issue at 6.25pm, Thursday, 3rd September 2009]**

Use merkaartor_split.aa merkaartor_split.3.ab merkaartor_split.ac to recreate the file (trac had problems when uploading the second part, so I have uploadet it again)
cat merkaartor_split.aa merkaartor_split.3.ab merkaartor_split.ac > merkaartor.icns
MD5 (merkaartor.icns) = 275a6678a277ecb77f87810a0d5c33aa

Sorry for the inconvenience

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 1.52pm, Friday, 18th September 2009]**

(In [17677]) ADD : Mac OS X icon (closes #2254)

**Comment by tomhughes** [comment]
**Author: aleksandar.topuzovic@gmail.com**
**[Added to the original trac issue at 10.10pm, Saturday, 19th September 2009]**

Note quite right in the last revision :(

```
Index: src/src.pro
===================================================================
--- src/src.pro (revision 17699)
+++ src/src.pro (working copy)
@@ -62,7 +62,7 @@
 }
 
 macx {
-       C_FILE = $$PWD/../Icons/merkaartor.icns
+       RC_FILE = $$PWD/../Icons/merkaartor.icns
 }
```

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 12.46pm, Sunday, 20th September 2009]**

(In [17720]) FIX: typo (closes #2254)
